lib/passwordValidator.ts

```diff
@@ -0,0 +1,13 @@
+export const validatePassword = (password: string): string => {
+	let errorMessage = '';
+	const passwordRegex = /(?=.*[0-9])/;
+
+	if (!password) {
+		errorMessage = 'Password cannot be empty';
+	} else if (password.length < 8) {
+		errorMessage = 'Password must be 8 characters long';
+	} else if (!passwordRegex.test(password)) {
+		errorMessage = 'Invalid password. Must contain one number';
+	}
+	return errorMessage;
+};
```

Comment:
```suggestion
export const validatePassword = (password: string): string => {
	const passwordRegex = /(?=.*\d)(?=.*[!@#$%^&*])(?=.*[A-Z])(?=.*[a-z]).{8,}/;

	if (!password) {
		return 'Password cannot be empty';
	} else if (password.length < 8) {
		return 'Password must be at least 8 characters long';
	} else if (!passwordRegex.test(password)) {
		return 'Invalid password. Must contain one number, one capital letter, one small letter and one special character (!@#$%^&*)';
	}
	return '';
};
```

Reply:
@debdutdeb After Implementing the suggested changes we will get ES Lint errors saying that the else after a return statement is redundant. I will try to clean this up in some other way.